library/connect-web/v0.0.10/Dockerfile

```diff
@@ -1,10 +1,12 @@
 # syntax=docker/dockerfile:1.4
-FROM node:16.17.0-bullseye-slim
+FROM node:16.17.0-bullseye-slim AS build
 ARG PLUGIN_VERSION
 RUN test -n "${PLUGIN_VERSION}"
 WORKDIR /app
 COPY --link package*.json .
 RUN npm ci
 
+FROM node:16.17.0-bullseye-slim
+COPY --link --from=build /app /app
```

Comment:
Dug in some more and found that `/root/.npm` had cache which differed between builds.